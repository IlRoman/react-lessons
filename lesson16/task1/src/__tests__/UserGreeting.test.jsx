import React from "react";
import UserGreeting from "../UserGreeting";
import { mount } from "enzyme";

describe("<UserGreeting />", () => {
    it("should render correctly user greeting", () => {
        const wrappedComponent = mount(<UserGreeting />);
        expect(wrappedComponent).toMatchSnapshot();
    });
});