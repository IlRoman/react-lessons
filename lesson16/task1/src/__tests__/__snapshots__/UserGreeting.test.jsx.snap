// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserGreeting /> should render correctly user greeting 1`] = `
<UserGreeting>
  <h1>
    Hello, user ðŸ˜Ž
  </h1>
</UserGreeting>
`;
